### PR TITLE
Scenario tweaking and more

### PR DESCRIPTION
**Pull request by Vel-Vel**
A few questions (sorry if they are bothersome. I feel like I need a series of guidelines of what to do and what not to do when translating):

When the "っ" is at the end of a Japanese word like "くすくすっ", may it be translated as "--", as in "Kusukusu--"; or may it be dropped, as in "Kusukusu"?

In the parts where the Japanese text is the same (e.g., when the player character wakes up to a girl doing some sexual act on him), should the translation be the same throughout, or may the text have different translations so that it would not be too repetitive? I got this from looking at some of the translations from the Koikatsu Party translation, which does just that.

### CODE REVIEW

**Review by ManlyMarco — COMMENTED** [review]
Everything looks good, but I'm not sure what to do with the zz_machineTranslation edits, I think they should be moved to normal translation files?

**Comment by Vel-Vel** [comment]
> Everything looks good, but I'm not sure what to do with the zz_machineTranslation edits, I think they should be moved to normal translation files?

So this is what we should do after sufficiently cleaning up the machine translation files. Yes. I'll get to it.

**Comment by Vel-Vel** [comment]
I moved the machine translation files to the normal translation files (not the communication translations yet).

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c06/00/2/translation.txt [hunk 1]
```diff
@@ -1,14 +1,13 @@
...
-//「私、本当に不勉強で……今までの貴方への接し方について、反省する点がいくつもございます」=
-//「い、いや、全然大丈夫だよ。俺もやりすぎた時とかあったと思うし」=
-//「ええ。全ての物事には限度というものがございますから、それだけは今後も気を付けていただきます」=
-//「ですが、貴方に対する理解は深まりました。至らない身ではございますが、今後ともよろしくお願いいたします」=
+「貴方の特徴についてお話がございます」=“I would like to speak to you about your traits.”
+「特徴？　どんなところ？」=“Traits?　What kind?”
+「破廉恥なところです」=“Your shamelessness.”
+「あっ、えっと……ごめんなさい」=“Ah, ummm... sorry.”
+「うふふ、謝る必要はございません。むしろ、私が貴方に謝らなくてはなりません」=“Ufufu, there's no need for you to apologize. Rather, I should be the one apologizing to you.”
+「昨晩、貴方の特徴について祖母に相談したのですが、それが殿方だということを教わりました」=“I spoke with my grandmother about your traits last night, and she taught me it's what men do.”
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c06/00/2/translation.txt:9]: Removed line
```suggestion
「え……？」=“Eh...?”
「昨晩、貴方の特徴について祖母に相談したのですが、それが殿方だということを教わりました」=“I spoke with my grandmother about your traits last night, and she taught me it's what men do.”
```

File: translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/00/2/translation.txt [hunk 2]
```diff
@@ -1,10 +1,10 @@
 //
 // Dumped for Koikatsu v5.1 (CharaStudio) by KK_TextDump v1.4.6.0
 //
-「ふふっ、君とのおしゃべりがこんなにも楽しいなんてね」=“Fufu--, I never thought I'd have this much fun chatting with you.”
-「俺も、まだまだしゃべり足りないよ」=“Me too, I still have lots I want to chat with you about.”
+「ふふっ、君とのおしゃべりがこんなにも楽しいなんてね」=“Fufu! I never thought I'd have this much fun chatting with you.”
+「俺も、まだまだしゃべり足りないよ」=“Me too, I still have lots I want to chat with you about.“
```
Comment (ManlyMarco) [inline translation/en/RedirectedResources/assets/abdata/adv/scenario/c00/00/2/translation.txt:5]: Wrong quote
```suggestion
「俺も、まだまだしゃべり足りないよ」=“Me too, I still have lots I want to chat with you about.”
```

**Review by ManlyMarco — APPROVED** [review]

**Pull request merged**